modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -382,4 +391,140 @@ describe('Ecdsa tss helper functions tests', function () {
     });
   });
 
+  describe('signing share parsers', function() {
+    let userSignShare: ECDSAMethodTypes.SignShare;
+    let bitgoSignShare: ECDSA.SignConvertRT;
+    let userGammaAndMuShare: ECDSAMethodTypes.CreateUserGammaAndMuShareRT;
+    let userOmicronAndDeltaShare: ECDSAMethodTypes.CreateUserOmicronAndDeltaShareRT;
+    let bitgoOmicronAndDeltaShare: ECDSAMethodTypes.CreateUserOmicronAndDeltaShareRT;
+    let userSignatureShare: ECDSAMethodTypes.SignatureShare;
+    before(async function() {
+      const txRequest = {
+        txRequestId: 'randomId',
+        unsignedTxs: [{ signableHex: 'TOO MANY SECRETS', serializedTxHex: 'randomhex2' }],
```

Comment:
i think we can use ecdsaFixtures for this instead of doing this before hook, fixtures already have mocks for most shares and signing is a bit expensive to perform